tedana/reporting/KappaRho_DynPlot.py

```diff
@@ -0,0 +1,578 @@
+# ---
```

Comment:
@javiergcas I think this file should be removed and its functionality refactored into the dynamic_figures.py file !

Reply:
Correct. Sorry I just commented below in the wrong comment for this question